cql3/restrictions/clustering_key_restrictions.hh

```diff
@@ -9,12 +9,20 @@
 #pragma once
 
 #include "cql3/expr/expression.hh"
+#include "schema.hh"
 
 namespace cql3 {
 namespace restrictions {
 
 // Restrictions containing only clustering key columns, extracted from the WHERE clause.
 class clustering_key_restrictions {
+    schema_ptr _table_schema;
+
+    expr::expression _clustering_restrictions;
```

Comment:
I don't see why the class is needed. Can't we just have an expression in analyzed_where_clause?